src/static/js/modules/loan-comparison/common.js

```diff
@@ -239,17 +240,32 @@ common.options = {
     'county': 'counties'
 }
 
+common.jumboTypes = {
+    'agency': {val: 'agency', label: 'Conforming jumbo'}, 
+    'jumbo': {val:'jumbo', label: 'Jumbo (non-conforming)'}, 
+    'fha-hb': {val:'fha-hb', label: 'FHA high-balance'}, 
+    'va-hb': {val:'va-hb', label: 'VA high-balance'}
+};
+
+common.norms = ['conf', 'fha', 'va'];
+
+common.jumboMessages = {
```

Comment:
Yes! And @stephanieosan suggested we use the county error message from rate checker, so that will simplify the jumbo messaging.